frontend/src/components/UserDetail/EmployeeDetail.tsx

```diff
@@ -127,6 +127,7 @@ const findEmployee = (
   const employee = DriverToEmployees(drivers).find(
     (employee) => employee.id === employeeId
   );
+  console.log(drivers);
```

Comment:
Delete! We don't want console.logs getting merged in, they should just be used for testing purposes.